purchase_lead_hour/models/product_supplierinfo.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ProductSupplierinfo(models.Model):
+    _inherit = "product.supplierinfo"
+
+    # In odoo core, this field is an Integer, which in some cases,
+    # could not be accurate enough.
+    delay = fields.Float()
+
+    def get_next_availability_date(self):
```

Comment:
```suggestion
    def _get_next_availability_date(self):
```

This should be private.
In odoo, public methods are exposed through xmlrpc
rule of thumb is make everything private unless you really want this